theia/api/views/imagery_request_view_set.py

```diff
@@ -11,14 +16,37 @@ class ImageryRequestViewSet(viewsets.ModelViewSet):
     serializer_class = ImageryRequestSerializer
 
     def create(self, request, *args, **kwargs):
-        copy = request.data.copy()
-        copy['bearer_token'] = request.session['bearer_token']
-        copy['refresh_token'] = request.session['refresh_token']
-        copy['bearer_expiry'] = request.session['bearer_expiry']
-
-        serializer = self.get_serializer(data=copy)
+        serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
-        self.perform_create(serializer)
 
-        headers = self.get_success_headers(serializer.data)
-        return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
\ No newline at end of file
+        with get_authenticated_panoptes(
+                request.session['bearer_token'],
+                request.session['refresh_token'],
```

Comment:
Not blocking - i don't think this user will have a refresh token in the session if you used the oauth implicit flow to get bearer tokens

Reply:
Good point; and we don't use it at the moment either